### ISSUE TITLE
Migrate to python 3

### ISSUE THREAD

**Issue by Seba-Allis** [allis-asso/allis-backup#4]

**Comment by Seba-Allis** [comment]
Break at startup:

``` python
Traceback (most recent call last):
  File "/usr/bin/backup.py", line 3, in <module>
    from fs import btrfs
  File "/usr/lib/python3/dist-packages/fs/__init__.py", line 3, in <module>
    from btrfs import btrfs
ImportError: No module named 'btrfs'
```

Need to master python class mechanism...

**Comment by Seba-Allis** [comment]
fix on commit e14ec82d35031669389421d3fb1e0b06e633026a
